flow-report/assets/styles.css

```
@@ -0,0 +1,145 @@
+.App {
+  display: grid;
+  grid-template-areas: "side report";
+  grid-template-columns: min-content auto;
+  min-height: 100vh;
+  max-width: 100vw;
+}
+.App a {
+  color: var(--color-gray-800);
+  text-decoration: none;
+}
+
+.Hbar {
+  height: 1px;
+  width: 100%;
+  background-color: var(--color-gray-300);
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.Sidebar {
+  border-right: 1px solid var(--color-gray-300);
+}
+
+.Sidebar_current {
+  background-color: var(--color-gray-200);
+  font-weight: bold;
+}
+
+.SidebarSectionTitle {
+  height: 16px;
+  font-size: 11px;
+  margin: 32px 16px 0px 16px;
+  color: var(--color-gray-700);
+}
+
+.SidebarHeader {
+  margin: 16px;
+}
+.SidebarHeader_title {
+  font-size: 16px;
+  font-weight: bold;
+  width: max-content;
+}
+.SidebarHeader_date {
+  font-size: 12px;
+  color: var(--color-gray-700);
+  width: max-content;
+}
+
+.SidebarRuntimeSettings > summary {
+  padding: 4px 16px;
+  list-style: none;
+  color: var(--color-gray-700);
+  font-size: 12px;
+}
+
+.SidebarSummary {
+  width: 100%;
+  display: flex;
+  padding: 8px 16px;
+  column-gap: 16px;
+  align-items: center;
+}
+
+.SidebarSummary_icon {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  align-items: center;
+  justify-content: center;
+}
+.SidebarSummary_icon::before {
+  content: '☰';
+  font-weight: normal;
+  color: var(--color-blue-A700);
+}
+
+.SidebarSummary_label {
+  font-size: 14px;
+}
+
+.SidebarFlowStep {
+  display: flex;
+  column-gap: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+.SidebarFlowStep_label {
+  grid-column: 2;
+  margin-top: 14px;
+  margin-bottom: 14px;
+  justify-self: left;
+  font-size: 12px;
+}
+.SidebarFlowStep_label.navigation {
+  font-size: 14px;
+}
+
+.SidebarFlowStep_icon {
+  grid-column: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 16px;
+}
+.SidebarFlowStep_icon_line {
+  width: 2px;
+  flex-grow: 1;
+  background-color: var(--color-blue-A700);
+}
+
+.FlowStepIcon {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  align-items: center;
+  justify-content: center;
+}
+.FlowStepIcon.navigation::before {
```

```suggestion
.FlowStepIcon--navigation::before {
```

WDYT? `navigation` as a standalone class isn't applied globally for particular effect, so if we're just using it for the icon, let's make that clear :)